.github/workflows/testing.yml

```diff
@@ -105,6 +179,8 @@ jobs:
         run: |
           source virtualenv/houston3.7/bin/activate
           pytest -s -v --gitlab-remote-login-pat "${{ secrets.GITLAB_REMOTE_LOGIN_PAT }}" --cov=./ --cov-report=xml
+        env:
+          SQLALCHEMY_DATABASE_URI: "postgresql://postgres:testing@localhost/postgres"
```

Comment:
Are we only running the tests with PostGRESQL in this one place on Ubuntu?  I'd prefer to run tests on a full matrix of [macOS, Ubuntu] X [Python 3.7, 3.8, 3.9] X [Sqlite, PostGRES]. 

As for CodeCov reporting, we can run them on one variant, probably (Ubuntu, 3.9, PostGRES), once the tests pass above.

Reply:
The "services" part doesn't work for macos so we can't use this way of setting up postgresql for macos, that's why I'm not including the postgresql tests for macos.

Reply:
I did a quick search and it looks like the only way to set up postgresql outside of github ... :/

Reply:
@bluemellophone Would these tests with the addition of different python versions and Sqlite as a DB be satisfactory for this change request or do we need to investigate further for a way to include macOS?

Reply:
Running PostGRES tests on Ubuntu only should be fine then, thank you @karenc for checking.  We will not be doing major deployments on macOS with PostGRES.

Reply:
Then this PR is ready, I will retract this change request and likely approve.

Reply:
i walked ben thru getting postgresql set up on his mac and it was foreign enough to me that i kinda agree with it not be necessary in terms of testing.

Reply:
The postgresql tests are failing after "invoke app.db.upgrade" so I'm looking into it.  One thing I can see is that `Submission.major_type` allows `test in the code but doesn't allow it in the migration.